app/services/force/event_subscriber_translate_service.rb

```diff
@@ -137,14 +123,19 @@ def process_event_values(listing_id, values)
           text_to_translate.push(listing[domain_key])
         end
       end
-      text_to_translate
+      [text_to_translate, values.keys]
```

Comment:
**question (non-blocking):** : `values.keys` doesn't change within this function, do we need to return it here or can we grab this value from the caller block?

Reply:
Good point, this is redundant code, happy to remove unnecessary return values and arguments